stories/Modal/Modal.stories.jsx

```diff
@@ -0,0 +1,62 @@
+import { useState } from 'react';
+import { Modal, Button } from '../../components';
+
+const Template = args => {
+  const [openModal, setOpenModal] = useState(false);
+  return (
+    <>
+      <Button onClick={() => setOpenModal(true)}>Open Modal</Button>
+      {openModal && (
+        <Modal onClose={() => setOpenModal(false)} {...args}>
+          <Modal.Content>
```

Comment:
Since this has to be the simplest example of modal, I believe the modal.content component is not needed

Reply:
If I remove modal.content it will lose formatting and look like this

![image](https://user-images.githubusercontent.com/37362664/197623507-2d1dd43e-9101-49e2-b508-6f56a1eea1f9.png)